tests/unit-tests/scene/test_basic_idle_hub.cpp

```diff
@@ -41,7 +41,8 @@ struct BasicIdleHub: Test
 {
     mtd::AdvanceableClock clock;
     mtd::FakeAlarmFactory alarm_factory{};
-    ms::BasicIdleHub hub{mt::fake_shared(clock), alarm_factory};
+    std::shared_ptr<ms::BasicIdleHub> hub{std::make_shared<ms::BasicIdleHub>(
+        mt::fake_shared(clock), alarm_factory)};
```

Comment:
If we do a line split like this, we generally break each argument onto a different line (either of the following would be preferred):
```cpp
    std::shared_ptr<ms::BasicIdleHub> hub{std::make_shared<ms::BasicIdleHub>(
        mt::fake_shared(clock),
        alarm_factory)};
```
```cpp
    std::shared_ptr<ms::BasicIdleHub> hub{
        std::make_shared<ms::BasicIdleHub>(mt::fake_shared(clock), alarm_factory)};
```